dist/codecov_ats.sh

```diff
@@ -114,28 +114,30 @@ if [[ -z $response && -n $INPUTS_OVERRIDE_BASE_COMMIT ]]; then
 	exit 1
 fi
 
-response=$(echo $response | sed 's/,//g')
-runner_options=$(echo $response | sed 's/^.*runner_options\": \[//' | sed 's/\].*$//')
-ats_tests_to_run=$(echo $response | sed 's/^.*ats_tests_to_run\": \[//' | sed 's/\].*$//')
-ats_tests_to_skip=$(echo $response | sed 's/^.*ats_tests_to_skip\": \[//' | sed 's/\].*$//')
-
-if [[ -z $runner_options ]]; then
-    say "$y==>$x Could not find 'runner_options', defaulting to '--cov-context=test'"
-    runner_options="--cov-context=test"
-fi
-
-if [[ -z $ats_tests_to_run ]]; then
-    say "$y==>$x No tests to run, picking random test"
-    ats_tests_to_skip_array=($ats_tests_to_skip)
-
-    if [[ -z $ats_tests_to_skip ]]; then
-        say "$y==>$x No tests to skip, running all tests"
-    else
-        ats_tests_to_run=${ats_tests_to_skip_array[ $RANDOM % ${#ats_tests_to_skip_array[@]} ]}
-    fi
+# Post process label-analysis response
+
+# Create directory to put result files
+mkdir codecov_ats
+# Export tests to run and tests to skip into respective files
+jq <<< "$response" '.runner_options + .ats_tests_to_run | @json' --raw-output > codecov_ats/tests_to_run.json
+jq <<< "$response" '.runner_options + .ats_tests_to_skip | @json' --raw-output > codecov_ats/tests_to_skip.json
+
+
+# Statistics on the test selection
+testcount() { jq <<< "$response" ".$1 | length"; }
+run_count=$(testcount ats_tests_to_run)
+skip_count=$(testcount ats_tests_to_skip)
+# Parse any potential errors that made ATS fallback to running all tests and surface them
+ats_fallback_reason=$(jq <<< "$response" '.ats_fallback_reason')
+if [ "$ats_fallback_reason" == "null" ]; then
+    ats_success=true
+else
+    ats_success=false
 fi
+tee <<< \
```

Comment:
what's the use of `tee` here do?

Reply:
sorry, I realize my comments are on `dist` not `src`

Reply:
`tee` is to put the results both in the summary of github actions (so it's easy for customers to know what happened with the ATS run) and in the file `codecov_ats/result.json`

Reply:
oh that's cool, I've never seen this syntax before